### DISCUSSION TITLE
CODEOWNERS file?

### DISCUSSION THREAD

**Discussion by spenserblack** [o2sh/onefetch#671] [Ideas]
Some of the collaborators of this project specialize in certain fields. For this reason, we might want to add a [`CODEOWNERS`](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file.
For example, @CephalonRho could be a code owner of `src/ui/image_backends/`, as they were the main force behind setting up support for images, and @Byron is definitely our `gitoxide` expert and should probably own code that handles git interactions.

**Comment by Byron** [comment]
Thanks for bringing this up! I think such a file would help to make explicit what's currently happening. I see myself as owning the following:

- upgrades of `gitoxide` to the latest version when relevant features make it necessary
- implement features in `gitoxide` if those are required in `onefetch`
- drive the transition to `gitoxide` to completion by implementing the necessary features in `gitoxide`
- support questions and issues related to `gitoxide`
